perfmetrics/scripts/run_e2e_tests.sh

```diff
@@ -145,6 +156,22 @@ function run_parallel_tests() {
   return $exit_code
 }
 
+function print_test_logs() {
+  readarray -t TEST_LOGS_ARRAY < "$TEST_LOGS_FILE"
+  rm "$TEST_LOGS_FILE"
+  for test_log_file in "${TEST_LOGS_ARRAY[@]}"
+  do
+    log_file=${test_log_file}
+    if [ -f "$log_file" ]; then
+      echo "=== Log for ${test_log_file} ==="
+      cat "$log_file"
+      echo "========================================="
+      # Remove the log file
+      rm "$log_file"
```

Comment:
nit: there is no harm in not removing the log file. incase someone ssh into the VM, they will be able to see the log files.

Reply:
Okay, If we want to keep it then I changed path to `/tmp` for not creating redundant files in current directory.

Reply:
please remove `rm` then ?

Reply:
Sorry missed. Done.